ballerina/tests/mock_test.bal

```diff
@@ -0,0 +1,181 @@
+import ballerina/http;
+import ballerina/oauth2;
+import ballerina/test;
+
+final string mockServiceUrl = "http://localhost:9090/crm/v3/objects/quotes";
+
+final Client mockClient = check new Client(config = {auth: {
+    clientId,
+    clientSecret,
+    refreshToken,
+    credentialBearer: oauth2:POST_BODY_BEARER 
+}}, serviceUrl = mockServiceUrl);
```

Comment:
This is not a `mockClient` but rather a client that connects to a mock service. Let's rename the variable accordingly.